tests/services/test_cluster.py

```diff
@@ -1,11 +1,13 @@
 import asyncio
 import itertools
 import sys
-import pytest
-from yapapi.services import Service, ServiceRunner
-from unittest.mock import Mock, patch
 from unittest import mock
+from unittest.mock import Mock, patch
+
+import pytest
```

Comment:
and here... I'm unsure if we do want to use this import sorting option...

Reply:
Default sections `FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER`
unittest is stdlib but pytest is thirdparty.

I can make it so everything is in a single group but i find it useful to have those groups most of the time